Flutter/pdf-viewer/sticky-notes.md

```diff
@@ -0,0 +1,139 @@
+---
+layout: post
+title: Sticky Note Annotations in Flutter PDF Viewer Widget | Syncfusion
+description: Learn here all about sticky note annotations in the Syncfusion Flutter PDF Viewer (SfPdfViewer) widget and more.
+platform: flutter
+control: SfPdfViewer
+documentation: ug
+---
+
+# Sticky Note Annotations in Flutter PDF Viewer Widget (Syncfusion)
+
+The sticky note annotation feature of `SfPdfViewer` allows you to add, remove, and modify sticky notes in a PDF document. This feature will help add comments or notes to specific parts of a document to clarify complex concepts, terms, or ideas. This section will cover the various functions available in `SfPdfViewer` for working with sticky note annotations.
```

Comment:
Can this be API linked? 
<img width="339" alt="image" src="https://github.com/user-attachments/assets/95d34b54-51b2-4451-ba82-f80a6afaab13">

Reply:
API url linked